Docs: New how-to sublevels (Diátaxis) (#10131)
readthedocs/readthedocs.org: docs/user/guides/build-troubleshooting.rst

Troubleshooting build errors
============================

.. include:: /shared/contribute-to-troubleshooting.rst

> The following is a list of common errors and resolutions encountered in the :doc:`build process </builds>`.

Reviewer: ```suggestion
This guide provides some common errors and resolutions encountered in the :doc:`build process </builds>`.
```
